### DISCUSSION TITLE
Queue settings do not take effect

### DISCUSSION THREAD

**Discussion by Apotato98** [apache/dolphinscheduler#12680] [Q&A]
![a03eb49ddee364fc8591124930d4168](https://user-images.githubusercontent.com/60845098/199643483-b640c11b-d4ca-45a1-b9fa-a83e9fdac600.jpg)
As shown in the figure, I have set up queues (non-default queues) for both tenants and users according to similar settings, but whether the user runs tasks or other users run tasks, the final queue used on yarn is still default. 
Besides, the version is 3.0.
The queue name on dolphinscheduler is the same as the queue name on yarn.

**Comment by Apotato98** [comment]
The queue name on dolphinscheduler is the same as the queue name on yarn.

**Comment by christianxiaobai** [comment]
I have the same problem
